app/src/main/java/io/gnosis/safe/ui/transactions/details/view/ValueItem.kt

```diff
@@ -9,7 +9,7 @@ import io.gnosis.safe.R
 import io.gnosis.safe.databinding.ViewTimeInfoItemBinding
 import timber.log.Timber
 
-class TimeInfoItem @JvmOverloads constructor(
```

Comment:
I am a bit confused by this name. Is this what we use for dates as well as other stuff too? Like a label + some value?

Reply:
You are absolutely right, that's why I have renamed it to ValueItem. Maybe LabeledValueItem would be more descriptive? The naming contest is officially open :)